### ISSUE TITLE
lolminer 1.39 http connection

### ISSUE THREAD

**Issue by clownel** [Lolliedieb/lolMiner-releases#903]
i'm not very good at it.. but.. i want to use lolminer for my rigs... atm i'm ussing a proxy ( eth-proxy by Atrides  on github ). The problem is when i'm trying to use lolminer, rigs connecting to my proxy.. but won`t start mining..i think is because it connecting to "stratum".  Atm i'm using t-rex miner, connecting to my proxy via http ( ./t-rex -a ethash -p http://proxyip:port.......)

lolminer start cmd is -algo ETHASH --pool PROXYip:PORT --user adress.workername.

Thx in advance !

**Comment by jgonzis** [comment]
> i'm not very good at it.. but.. i want to use lolminer for my rigs... atm i'm ussing a proxy ( eth-proxy by Atrides on github ). The problem is when i'm trying to use lolminer, rigs connecting to my proxy.. but won`t start mining..i think is because it connecting to "stratum". Atm i'm using t-rex miner, connecting to my proxy via http ( ./t-rex -a ethash -p http://proxyip:port.......)
> 
> lolminer start cmd is -algo ETHASH --pool PROXYip:PORT --user adress.workername.
> 
> Thx in advance !

You are right connection is stratum, why don't you try with socks5? if that helps you on you configuration

**Comment by clownel** [comment]
well.. i wish that connection to be http...not sure where to find a good socks5 proxy...

**Comment by GTreaper** [comment]
> well.. i wish that connection to be http...not sure where to find a good socks5 proxy... and my proxy accept just http connections :(

I'm facing the same issue, also do my connection via eth proxy and can't connect. Did you figure it out or no?
